src/Growthbook.php

```diff
@@ -100,7 +102,9 @@ public function __construct(array $options = [])
             "requestFactory",
             "decryptionKey",
             "stickyBucketService",
-            "stickyBucketIdentifierAttributes"
+            "stickyBucketIdentifierAttributes",
+            "savedGroups",
+            "encryptedSavedGroups"
```

Comment:
Are `encryptedSavedGroups` being handled anywhere?

Reply:
Yes, here -- https://github.com/growthbook/growthbook-php/blob/av/sdk-updates/src/Growthbook.php#L1069

One thing I want to do is separate `loadFeatures` a bit more, because right now it is named `loadFeatures` but it is also handling savedGroups.

Reply:
In the javascript SDK, we renamed it to `init` (we kept around `loadFeatures` as an alias for backwards compatibility)

Reply:
Done -- created `initialize`.